azext_iot/central/_help.py

```diff
@@ -397,6 +397,21 @@ def _load_central_devices_help():
             --device-id {deviceid}
     """
 
+    helps[
+        "iot central device purge-messages"
```

Comment:
If the goal is to support more c2d commands in the future, consider an additional command group encompassing them to improve organization and reduce "verb-noun" commands. For example `az iot central device c2d-message purge`.

There is an existing c2d message command group in `az iot device`, though its focused for IoT Hub interaction which illustrates this idea. Pasting screenshot here for convenience:

 
![image](https://user-images.githubusercontent.com/28658112/147999095-ba692222-414d-4a46-8687-f29ff3504652.png)